packages/babel-plugin-transform-classes/src/loose.js

```diff
@@ -64,4 +72,22 @@ export default class LooseClassTransformer extends VanillaTransformer {
       return true;
     }
   }
+
+  _processSetGet(node, path) {
+    const nonConstructorMethod = path.container.filter(containedNode => {
+      return containedNode.kind !== "constructor";
+    });
+    if (nonConstructorMethod.length === 1) {
+      this.body.push(
+        objectDefineProperty({
```

Comment:
Github seems to have lost my comment, but:

This creates separate descriptors for `get` and `set`, but they have to be the _same_ descriptor, otherwise the previously declared descriptor will be lost.

Reply:
Thanks a lot for the review. I have done all the other changes, and I am working on this one right now, but if you have any suggestions on best way to tackle this?

Reply:
as stated under other comments here - this is actually not true, descriptors will be merged, but anyway in this part of the code from what i understand we wouldnt have to worry about this case anyway as its running only for `if (nonConstructorMethod.length === 1)` case

Reply:
@Kovensky @Andarist I think we should not merge the descriptors because of issue it might create with computed properties 
```js
const foo = () => 'foo'
class MyClass {
  get [foo()]() { this._foo || 'foo' }
  set [foo()](val) { this._foo = val }
}
```

Reply:
@Andarist slack chat link https://babeljs.slack.com/archives/C062RC35M/p1515617623000644?thread_ts=1515589048.000272&cid=C062RC35M

Reply:
we can merge non-computed ones though